packages/gatsby-transformer-screenshot/src/gatsby-node.js

```diff
@@ -114,6 +118,10 @@ const createScreenshotNode = async ({
       createNodeId,
     })
 
+    if (!fileNode) {
```

Comment:
`fileNode` can sometimes be null. My current guess is that the lambda function could return a URL for the new file on s3 but perhaps sometimes it's not there yet?